expression/builtin_cast_vec_test.go

```diff
@@ -37,6 +37,7 @@ var vecBuiltinCastCases = map[string][]vecExprBenchCase{
 		},
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETInt}},
 		{retEvalType: types.ETDecimal, childrenTypes: []types.EvalType{types.ETDatetime}},
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETReal}},
```

Comment:
We may add a dataGenerator for this test case.

Reply:
What's wrong with using the default dataGenerator?

Reply:
Never mind, it's ok here